virttest/utils_test/__init__.py

```diff
@@ -269,7 +269,7 @@ def get_time(session, time_command, time_filter_re, time_format):
         finally:
             locale.setlocale(locale.LC_TIME, loc)
 
-        output = session.cmd_output_safe(time_command)
+        output = session.cmd_output_safe("LC_TIME=C %s" % time_command)
```

Comment:
Is there some reason to do hardcoding here?

Reply:
Thanks, because `locale.setlocale(locale.LC_TIME, "C")` didn't work for guest, so add hardcode here. It affects time format only, so I think it is ok.